strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/ArtifactEntry.java

```diff
@@ -38,6 +40,8 @@
     @ManyToMany(targetEntity = ArtifactTagEntry.class)
     private Set<ArtifactTag> tagSet;
 
+    private Map<String, String> checkSumsMap;
```

Comment:
@ogryniuk I also suggest to initialize the property with `new HashMap<>()` instance here, instead of having `Optional.ofNullable` logic in getter method.

Reply:
@sbespalov Another collection, Set, is also initialised through Optional. Will not following your suggestion break the consistency of the class where one variable (Set) would be declared in the getter and another one (Map) - immediately after declaration?

Reply:
Right, I had not noticed that field actually initialized in getter as well. 
Good catch @ogryniuk, thanks :)